iot_verifier/src/poc.rs

```diff
@@ -590,11 +598,19 @@ fn verify_witness_data(beacon_data: &Vec<u8>, witness_data: &Vec<u8>) -> Generic
     Ok(())
 }
 
-fn calc_fspl(tx_power: i32, freq: u64, distance: f64, gain: i32) -> f64 {
-    let gt = 0.0;
-    let gl = gain as f64 / 10.0;
-    let fpsl = (20.0 * (4.0 * PI * distance * (freq as f64) / C).log10()) - gt - gl;
-    (tx_power as f64) - fpsl
+fn calc_expected_rssi(
+    conducted_tx_power_dbm: i32,
+    freq: u64,
+    distance_mtrs: u32,
+    beaconer_gain_dbm: i32,
```

Comment:
Can you rename these parameters to:

`beaconer_gain_ddb` 
`witness_gain_ddb`

Reply:
fixed: 8b51e3bad7b345bf2f433bb9bae825497a795151